submissions/SamVal007/task-JS-dom/dom.js

```diff
@@ -0,0 +1,148 @@
+      "In the 2018–19 snooker season, he won his first Masters, defeating Ronnie O'Sullivan 10–4 in the final, and completed his Triple Crown by claiming his first world title at the 2019 World Snooker Championship, defeating Higgins 18–9. In doing so, he became the first player to win over £1 million in prize money in a single season. In the 2019–20 snooker season, he won six ranking tournaments, setting a new record for the most ranking titles in one season.",
+      "A prolific break-builder, Trump has compiled more than 750 century breaks in professional competition, making him the fourth player ever to reach the 750-century mark. In the 2019–20 season, he became the second player after Neil Robertson to compile 100 century breaks in a single season. Trump finished the season on 102 centuries, one fewer than Robertson's record 103."
+    ]
+  },
+  {
+    name: "Neil Robertson",
+    born: "11 February 1982 (age 38) Melbourne, Victoria, Australia",
+    currentRanking: "2 (as of 21 December 2020)",
+    careerWinnings: "£5,318,112",
+    picture: "./img/Neil Robertson.jpg",
+    descr: [
+      "Neil Robertson (born 11 February 1982) is an Australian professional snooker player. He made his first breakthrough into the top professional ranks in the 2006–07 season. He won the 2010 World Championship and was the world number one later in the same year, a ranking that he attained again in 2013 and 2014. On 10 November, Neil Robertson won the 2019 Champion of Champions tournament for the second time in his career and secured his 19th ranking event.",
+      "Robertson is the only Australian to have won a ranking event, and was undefeated in his first six televised finals. Robertson is also one of thirteen players to win both the world and UK titles, and one of eleven to win the Triple Crown of World Championship, UK Championship and Masters. As a prolific break-builder, Robertson has compiled more than 750 century breaks in professional competition. During the 2013–14 season he became the first player to make 100 centuries in a single season.",
+      "Robertson is the most successful player from outside the United Kingdom in the sport's history. He plays left-handed.",
+      "In June 2016, he became ambassador of electronic snooker simulator app Snooker Live Pro. He was an avid gamer but gave up the hobby in April 2017, believing he was spending too much time playing games and it was affecting his snooker form."
+    ]
+  },
+]
+
+const playerPhoto = document.querySelector('.photo');
+const playerName = document.querySelector('.player-name');
+const playerBorn = document.querySelector('.born');
+const playerCurrentRanking = document.querySelector('.current-ranking');
+const playerCareerWinnings = document.querySelector('.career-winnings');
+const playerDescription = document.querySelector('.descr');
+const nav = document.querySelector('nav');
```

Comment:
Does the nav have a certain role in the application? I would try to find a better explanation for why it is here

Reply:
Basically, dont know where to put it better:)
LEt's put it in the top? It's main element i would say.

Reply:
I think `menuContainer`, `navigationNode` or something like this may be used, so that variable name explains why we need it. In terms of your application logic it does not matter if the DOM element is a nav or a div, that's why I was confused